conda-recipes/llvmdev_manylinux/build.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# based on https://github.com/AnacondaRecipes/llvmdev-feedstock/blob/master/recipe/build.sh
```

Comment:
PS: Maybe include a link to the commit hash or version here?

Reply:
This file is a copy of `../llvmdev/build.sh` with minor changes. I'll add file level comments to document this.